package.json

```diff
@@ -20,7 +24,8 @@
     "test:coverage": "nyc --reporter=text mocha --exit --timeout 4000 --require ts-node/register --reporter spec \"./spec/**/*.spec.ts\"",
     "watch-test": "mocha --require ts-node/register --watch --reporter spec \"./spec/**/*.spec.ts\"",
     "prepublishOnly": "echo \"Do not run publish directly, run publisher\" && exit 1",
-    "publisher": "publisher"
+    "publisher": "publisher",
+    "prepare": "husky install"
```

Comment:
just to confirm, the `prepare` here is only run when we do a local `npm install` within this package, and not when someone else does `npm install sqlectron-db-core`?

Reply:
Right